modules/light-clients/08-wasm/internal/ibcwasm/expected_interfaces.go

```diff
@@ -51,6 +51,24 @@ type WasmEngine interface {
 		deserCost wasmvmtypes.UFraction,
 	) ([]byte, uint64, error)
 
+	// Migrate migrates an existing contract to a new code binary.
+	// This takes storage of the data from the original contract and the code hash of the new contract that should
+	// replace it. This allows it to run a migration step if needed, or return an error if unable to migrate
+	// the given data.
+	//
+	// MigrateMsg has some data on how to perform the migration.
```

Comment:
```suggestion
	// MigrateContract has some data on how to perform the migration.
```

is that what was intended?

Reply:
yeah intended. It is talking about the payload